### ISSUE TITLE
Errors occur when running docker-compose up

### ISSUE THREAD

**Issue by qqaatw** [litespeedtech/ols-docker-env#27]
When I run docker-compose up, these errors occur:

ERROR: for 8e936524d694_ols-docker-env_litespeed_1  Cannot start service litespeed: Bind mount failed: '/ols-docker-env/lsws/conf' does not exists

ERROR: for litespeed  Cannot start service litespeed: Bind mount failed: '/ols-docker-env/lsws/conf' does not exists
ERROR: Encountered errors while bringing up the project.

**Comment by Code-Egg** [comment]
Hi, I just did a fresh setup, no such issue. Is it a fresh installation, or did you change anything? May I know if /ols-docker-env/lsws/conf folder exists?

**Comment by qqaatw** [comment]
Yes, I modified the compose file to specify an IP address for OLS, and the conf folder doesn't exist.
Is it the reason causing these errors?

[docker-compose.zip](https://github.com/litespeedtech/ols-docker-env/files/6262425/docker-compose.zip)

Thanks.

**Comment by Code-Egg** [comment]
Hi, 

I created macvlan network on top of eth0, command `docker-compose up` has no error shows and conf folder does exist. You might also want to take care of the network for MySQL so litespeed can connect to it after you figure out the conf gone issue.

**Comment by qqaatw** [comment]
Hi, 

After manually creating `lsws/conf` and `lsws/admin-conf` folders, it works!
It would be better for new users of OLS by adding some instructions into README or adding two empty folders into the repo.

Regards.

**Comment by Code-Egg** [comment]
HIHI, 
You shouldn't need to do it manually, because it should be auto-generated when you docker-compose up successfully. Feel free to try it again with/without network added and see if the folder generated.

**Comment by qqaatw** [comment]
Hi,

I figured out that the reason might be the docker package of Synology NAS, bind errors will occur when specified folders don't exist. I tested on another machine, which has ubuntu 18.04 installed, it could simply run `docker-compose up` command, then the folders were automatically generated without manually creating it first.

I think some users who want to install OLS on their Synology NAS may encounter the same problem as mine.

Regards.
